src/html/classic/ng/src/web/components/loading/loading.js

```diff
@@ -23,18 +24,40 @@
 
 import React from 'react';
 
-import _ from 'gmp/locale.js';
+import glamorous from 'glamorous';
+import {css} from 'glamor';
 
 import PropTypes from '../../utils/proptypes.js';
 
+const Loader = glamorous.div({
+    border: '16px solid #c8d3d9',
+    borderTop: '16px solid #66c430',
+    borderRadius: '50%',
+    width: '120px',
+    height: '120px',
+    fontWeight: 'bold',
+    animation: `${css.keyframes({
+      '0%': {transform: `rotate(0deg)`},
+      '100%': {transform: `rotate(360deg)`},
+    })} 2s linear infinite`,
+});
+
+const Center = glamorous.div({
+  display: '-webkit-flex',
+  display: 'flex',
+  width: '100%',
+  justifyContent: 'center',
+  paddingTop: '100px',
```

Comment:
Feels a bit wrong to use a "random" padding-top of 100px. Where does it come from?